db/db_wal_test.cc

```diff
@@ -1007,7 +1007,7 @@ TEST_F(DBWALTest, AvoidFlushDuringRecovery) {
   Reopen(options);
   ASSERT_EQ("v11", Get("foo"));
   ASSERT_EQ("v12", Get("bar"));
-  ASSERT_EQ(2, TotalTableFiles());
+  ASSERT_EQ(3, TotalTableFiles());
```

Comment:
@yiwu-arbug can you confirm this change? It seems that the test intended a force flush with setting allow_2pc but it was not effective due to absence of allow_2pc in the copy-constructor. If the force flush was intended the total number of sst files should be 3 as it is now after this patch.

Reply:
true, thanks for fixing.